foundry.toml

```diff
@@ -0,0 +1,50 @@
+[profile.default]
+  src = 'contracts'
+  out = 'out'
+  libs = ['node_modules', 'lib']
+  test = 'test'
+  cache_path  = 'cache_forge'
+  evm_version = "shanghai"
+  optimizer = true
+  optimizer_runs = 10_000_000
+  solc_version = "0.8.18"
+  verbosity = 3
+
+[profile.ci]
+  fuzz = { runs = 5000 }
+  invariant = { runs = 1000 }
+
+[profile.lite]
+  fuzz = { runs = 50 }
+  invariant = { runs = 10 }
+  # Speed up compilation and tests during development.
+  optimizer = false
+
+[invariant]
+  call_override = false
+  depth = 100
+  dictionary_weight = 80
+  fail_on_revert = false
+  include_push_bytes = true
+  include_storage = true
+  optimizer = false
+  runs = 25
+
+[fmt]
+  bracket_spacing = false
+  int_types = "long"
+  line_length = 120
+  multiline_func_header = "attributes_first"
+  number_underscore = "thousands"
+  quote_style = "double"
+  single_line_statement_blocks = "multi"
+  tab_width = 4
+  wrap_comments = true
+  ignore = [
```

Comment:
curious, why mark these as ignore?

Reply:
A number of the original contracts got modified by "forge fmt" and adding them to the ignore list allows them to not show up as large diffs on the PR.